index.pug

```diff
@@ -16,23 +18,23 @@ html(lang="en")
 		-const linkHuge = "https://www.hugeinc.com/"
 		-const images= [{ name: "Program", image: './assets/program.jpg'},{ name: "Talent", image: './assets/talent.jpg'},{ name: "Tech", image: './assets/tech.jpg'}]
 
-		header.topbar
-			h1= heading[0]
-			.main-menu
-				figure
-					a(class="main-menu__link" href=linkHuge) 
-						img( class="main-menu__logo" src='./assets/Huge_logo_1.jpeg' alt="Huge logo. The link will take you to the official website of the company.")
-					nav
-						ul: each option in mainMenu
-							li: a(href= option)= option
+		header.header
+			h1(class="header__title") Make huge moves
+			figure
+				a(class="header__link" href=linkHuge)
+					img(class="header__logo" src='./assets/Huge_logo_obscuro.png' alt="Huge logo")
 			include regionalMenu
-		main
+		nav.navbar
+			ul: each option in mainMenu
+				li: a(class="navbar__items" href= option)= option
+			
+		main.main
```

Comment:
If your class name is the same name as the element, in this case `main` = `main` there is no need for using a class here (the same goes for other elements with the same approach)

Reply:
Bloque: Es esa parte independiente en la cual se escrutará nuestro proyecto y donde se verán contenidos todos los elementos de este. es decir, esos contenedores principales que nos ayudaran a dividir las partes de nuestro documento y que son fácilmente identificadas. Ejemplos de esto serian header, footer, contenedor principal, ítems y demás. En realidad, depende de ti y tu forma de estructurar resaltando que los bloque serán los contenedores de los elementos y así mismo deben ser nombrados de forma coherente.
Ejemplo de cómo nombrar bloques:

.footer

Corrijeme si me equivoco pero las etiquetas semanticas como header, footer, main.... son bloques y por ende se maneja desde css con clases en lugar de la etiqueta